source/mir/glas/common.d

```diff
@@ -0,0 +1,66 @@
+/++
+$(H2 General Matrix-Vector Multiplication)
+
+$(SCRIPT inhibitQuickIndex = 1;)
+
+This is a submodule of $(LINK2 mir_glas.html, mir.glas).
+
+License: $(LINK2 http://boost.org/LICENSE_1_0.txt, Boost License 1.0).
+
+Authors: Ilya Yaroshenko
++/
+module mir.glas.common;
+
+/// Conjugation type
+enum Conjugation
+{
+    /// Pseudo code for `gemm` is `c[i, j] += a[i, k] * b[k, j]`
+    none,
+    /// Pseudo code for `gemm` is `c[i, j] -= a[i, k] * b[k, j]`, for internal use only
+    sub,
+    /// Pseudo code for `gemm` is `c[i, j] += conj(a[i, k]) * b[k, j]`
+    conjA,
+    /// Pseudo code for `gemm` is `c[i, j] += a[i, k] * conj(b[k, j])`
+    conjB,
```

Comment:
Imho conjA, conjB and conjC are pretty bad names for a public API, because you always have to lookup how they are defined. I still have to understand their meaning, but couldn't something a bit more explicit like conjFirst, conjRow, ... work?